coderd/workspaces_test.go

```diff
@@ -36,8 +36,9 @@ func TestWorkspace(t *testing.T) {
 
 	t.Run("OK", func(t *testing.T) {
 		t.Parallel()
-		client := coderdtest.New(t, &coderdtest.Options{IncludeProvisionerDaemon: true})
+		client, _, api := coderdtest.NewWithAPI(t, &coderdtest.Options{IncludeProvisionerDaemon: true})
 		user := coderdtest.CreateFirstUser(t, client)
+		authz := coderdtest.AssertRBAC(t, api, client)
```

Comment:
See this unit test to see how to asset an rbac action was checked. 

This is a very basic check on 1 `GET`, but shows how you can assert certain rbac checks are run with the given client as the subject.

Reply:
This is a nice feature to have, but given that it essentially duplicates what we already codify in our authz_querier, we should probably use it sparingly.